docs/src/administration/users.md

```diff
@@ -10,40 +10,44 @@ description: |
 
 ## User roles
 
-At the project level:
+{{< note theme="warning" title="Important" >}}
+We are deprecating granting user roles on individual environments. You should use environment types instead.
+{{< /note >}}
 
-* **Project Administrator** - A project administrator can change settings and execute actions on any environment.
-* **Project Viewer** - A project reader can view all environments within a project but cannot execute any actions on them.
+User roles are defined per environment types (Production, Staging, Development):
 
-A Project Reader can have a specific role on different environments. At the environment level:
+* **Administrator** - Administrator can change settings and execute actions on all environment of this type.
+* **Contributor** - Contributor can push code and branch all environments of this type.
+* **Viewer** - Viewer can view all environments of this type.
 
-* **Environment Administrator** - An environment administrator can change settings and execute actions on this environment.
-* **Environment Contributor** - An environment contributor can push code to this environment and branch the environment.
-* **Environment Viewer** - An environment reader can only view this environment.
+User roles can also be granted to the entire project:
+
+* **Project Administrator** - A project administrator can change settings and execute actions on all environments of the project.
+* **Project Viewer** - A project viewer can view all environments of the project.
 
 {{< note theme="warning" title="Important" >}}
-After a user is added to (or deleted from) an environment, it will be automatically redeployed, after which the new permissions will be fully updated.
+After a user is added to (or removed from) an environment type, all environments of this type will be automatically redeployed, after which the new permissions will be fully updated.
```

Comment:
This is not currently the case as discussed here; https://platformsh.slack.com/archives/CQH0G037H/p1623166251188600